network/ServerNetworking.h

```diff
@@ -227,8 +227,9 @@ class PlayerConnection :
     const std::string& ClientVersionString() const;
 
     /** Returns true iff the client's specified version string matches the
-      * version string of this server process. */
-    bool ClientVersionStringMatchesThisServer() const;
+      * version string of this server process so the server will enable binary serialization if it
+      * is allowed. */
+    bool IsBinarySerializationEnabled() const;
```

Comment:
name a bit misleading... enabling not the same as using, as it needs to be enabled and the version strings need to match for this to return true.